src/pages/Compare/index.spec.tsx

```diff
@@ -1361,4 +1380,133 @@ describe(__filename, () => {
       `Compare ${name}: ${baseVersionId}...${headVersionId}`,
     );
   });
+
+  describe('updateSelectedPath', () => {
+    const setUpForPathUpdateAndRender = ({
+      baseVersionId,
```

Comment:
`baseVersionId` is not actually used in any of the tests, so you could just set it to a value inside this function. Because `nextUniqueId()` is guaranteed to be unique, you could just pass `baseVersionId: nextUniqueId()` in the call to `loadDiffAndRender` and then each test wouldn't have to pass a value for `baseVersionId` into this function.

Reply:
My original thought is to explicitly set `headVersionId > baseVersionId` in each tests, so that the `loadData` function will not return at https://github.com/mozilla/addons-code-manager/blob/fbd1c848f71717e61341180318c21b4eba6c0eb8/src/pages/Compare/index.tsx#L118
 I moved this logic to the helper, removed the parameter `baseVersionId`, and set `baseVersionId = headVersionId -1` in helper. It probably is better.